docs/user/azure/install_upi_azurestack.md

```diff
@@ -0,0 +1,606 @@
+apiVersion: v1
+kind: Secret
+metadata:
+    name: installer-cloud-credentials
+    namespace: openshift-image-registry
+stringData:
+  azure_subscription_id: <subscription-id>
+  azure_client_id: <client-id>
+  azure_client_secret: <secret>
+  azure_tenant_id: <tenant>
+  azure_resource_prefix: <$INFRA_ID>
+  azure_resourcegroup: <$RESOURCE_GROUP>
+  azure_region: <$REGION>
+apiVersion: v1
+kind: Secret
+metadata:
+    name: cloud-credentials
+    namespace: openshift-ingress-operator
+stringData:
+  azure_subscription_id: <subscription-id>
+  azure_client_id: <client-id>
+  azure_client_secret: <secret>
+  azure_tenant_id: <tenant>
+  azure_resource_prefix: <$INFRA_ID>
+  azure_resourcegroup: <$RESOURCE_GROUP>
+  azure_region: <$REGION>
+apiVersion: v1
+kind: Secret
+metadata:
+  name: azure-cloud-credentials
+  namespace: openshift-machine-api
+stringData:
+  azure_subscription_id: <subscription-id>
+  azure_client_id: <client-id>
+  azure_client_secret: <secret>
+  azure_tenant_id: <tenant>
+  azure_resource_prefix: <$INFRA_ID>
+  azure_resourcegroup: <$RESOURCE_GROUP>
+  azure_region: <$REGION>
+```
+
+## Create ignition configs
+
+Now we can create the bootstrap ignition configs:
+
+```console
+$ openshift-install create ignition-configs
+INFO Consuming Openshift Manifests from target directory
+INFO Consuming Worker Machines from target directory
+INFO Consuming Common Manifests from target directory
+INFO Consuming Master Machines from target directory
+```
+
+After running the command, several files will be available in the directory.
+
+```console
+$ tree
+.
+├── auth
+│   └── kubeconfig
+├── bootstrap.ign
+├── master.ign
+├── metadata.json
+└── worker.ign
+```
+
+## Create The Resource Group
+
+Use the command below to create the resource group in the selected Azure region:
+
+```sh
+az group create --name $RESOURCE_GROUP --location $AZURE_REGION
+```
+
+## Upload the files to a Storage Account
+
+The deployment steps will read the Red Hat Enterprise Linux CoreOS virtual hard disk (VHD) image and the bootstrap ignition config file
+from a blob. Create a storage account that will be used to store them and export its key as an environment variable.
+
+```sh
+az storage account create -g $RESOURCE_GROUP --location $AZURE_REGION --name ${CLUSTER_NAME}sa --kind Storage --sku Standard_LRS
+export ACCOUNT_KEY=`az storage account keys list -g $RESOURCE_GROUP --account-name ${CLUSTER_NAME}sa --query "[0].value" -o tsv`
+```
+
+### Copy the cluster image
+
+In order to create VMs, the RHCOS VHD must be available in the Azure Stack
+environment. The VHD should be downloaded locally, decompressed, and uploaded to a
+storage blob.
+
+First, download and decompress the VHD, note that the decompressed file is 16GB:
+
+```sh
+$ export COMPRESSED_VHD_URL=$(curl -s https://raw.githubusercontent.com/openshift/installer/release-4.9/data/data/rhcos-amd64.json | jq -r '(.baseURI + .images.azurestack.path)')
+$ wget $COMPRESSED_VHD_URL
```

Comment:
I would have a tendency to use `curl` instead of `wget` as it might not be on all systems by default if I remember correctly.

Reply:
That might be debatable, but I think we should let the reader know that he might need to install either `wget` or `curl`